index.html

```diff
@@ -2,65 +2,71 @@
-            <a id="logo" href="https://brickhack.io">
-                <img src="assets/logo.svg" alt="BrickHack 7 Logo" />
-            </a>
-            <a class="link" href="https://brickhack7.devpost.com/">DEVPOST</a>
-            <a class="link" href="./gallery.html">GALLERY</a>
-        </div>
-        <div class="right">
-            <a id="login" href="https://apply.brickhack.io/">LOGIN</a>
-            <a id="mlh-badge" href="https://mlh.io/seasons/na-2021/events?utm_source=na-hackathon&amp;utm_medium=TrustBadge&amp;utm_campaign=2020-season&amp;utm_content=gray">
-                <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2021/mlh-trust-badge-2021-gray.svg"  alt="Major League Hacking 2021 Hackathon Season" />
-            </a>
-        </div>
-    </nav>
-    <section id="end-main" class="section-pad">
-        <div id="thank-you-container">
-            <h1 class="end-h1">Thank you for attending BRICKHACK 7</h1>
-            <p>We hope to see you next year!</p>
-            <a href="./event.html">View the full event site</a>
+    <section id="main">
+
+        <img id="logo" src="./resources/pixel-logo-real.png" alt="BrickHack 8 logo. Two stripes, left orange and right blue, in a parallelogram with a white border.">
+
+        <div class="window">
+            <div class="title-bar">
+                <div class="title-bar-text">BrickHack 8</div>
+                <div class="title-bar-controls">
+                    <button aria-label="Minimize"></button>
+                    <button aria-label="Maximize"></button>
+                    <button aria-label="Close"></button>
+                </div>
+            </div>
+            <div class="window-body">
+                <p>BrickHack 8 is returning in early 2022!</p>
+                <div class="field-row-stacked">
+                    <label for="prereg-email">Enter your e-mail to stay updated.</label>
+                    <div class="field-row">
+                        <input id="prereg-email" type="text" placeholder="hacker@email.com">
+                        <button>Submit</button>
+                    </div>
+                </div>
+            </div>
         </div>
-        <div id="ricky-container">
-            <!-- <video playsinline autoplay muted loop poster="" id="ricky-wave">
-                <source src="./assets/post-event/closing.mp4" type="video/mp4">
-            </video> -->
-            <img src="./assets/post-event/closing-crop.gif" />
+        <div id="badges">
+            <img src="./assets/bh8/macmade-blkwtbrdrspin.gif" alt="Made with Macintosh badge">
+            <img src="./assets/bh8/adobesvg.gif" alt="This side is SVG powered badge">
+            <img src="./assets/bh8/hackerpowered.gif" alt="Hacker powered badge">
         </div>
     </section>
+    <footer>
+        <p>Interested in sponsoring? <span class="text-break">E-mail <a href="mailto:sponsorship@coderit.org">sponsorship@coderit.org</a></span></p>
```

Comment:
This should be using a brickhack.io email instead of a coderit.org one, if at all possible.